requirements.txt

```diff
@@ -68,3 +68,4 @@ pytest-asyncio==0.20.3
 pytest-cov==4.0.0
 pytest-timeout==2.1.0
 pytest-xdist==3.1.0
+sqlalchemy-stubs==0.4
```

Comment:
I cannot see you use this ?

We should not use a sub, unless it is absolutely needed.

Reply:
```
pymodbus/datastore/database/sql_datastore.py:4: error: Skipping analyzing "sqlalchemy": module is installed, but missing library stubs or py.typed marker  [import]
pymodbus/datastore/database/sql_datastore.py:5: error: Skipping analyzing "sqlalchemy.types": module is installed, but missing library stubs or py.typed marker  [import]
pymodbus/datastore/database/sql_datastore.py:6: error: Skipping analyzing "sqlalchemy.pool": module is installed, but missing library stubs or py.typed marker  [import]
pymodbus/datastore/database/sql_datastore.py:7: error: Skipping analyzing "sqlalchemy.schema": module is installed, but missing library stubs or py.typed marker  [import]
pymodbus/datastore/database/sql_datastore.py:8: error: Skipping analyzing "sqlalchemy.sql": module is installed, but missing library stubs or py.typed marker  [import]
pymodbus/datastore/database/sql_datastore.py:9: error: Skipping analyzing "sqlalchemy.sql.expression": module is installed, but missing library stubs or py.typed marker  [import]
```

There are missing type stubs for `pyserial` also, available via `pip install types-pyserial` but I'm not sure if they're correct right now: https://github.com/python/typeshed/pull/9722